assets/output.html.erb

```diff
@@ -70,14 +70,13 @@
     line-height: 2.2rem;
     margin: 0 0 0.5em;
     }
-    .information .infobox ul {
```

Comment:
This style section was unused (there were no lists in `.information .infobox`), so I removed it.